### PR TITLE
feat: adding version metadata to be able to specify kafka version

### PR DESCRIPTION
**Pull request by Dorn-**
Signed-off-by: Flavien Chantelot <contact@flavien.dev>

Add a new metadata `version` to let the user specify his kafka version. The default option is setted to 1.0.0 to avoid any breaking changes. 
### Checklist

- [X] Commits are signed with Developer Certificate of Origin (DCO)
- [X] A PR is opened to update the documentation on [our docs repo](https://github.com/kedacore/keda-docs)

Related to https://github.com/kedacore/keda/pull/1866

### CODE REVIEW

**Comment by tomkerkhove** [comment]
I will wait until the feature is merged, but thanks for the contribution!

**Review by zroubalik — APPROVED** [review]

**Pull request merged**
